src/frontend-samples/multi-viewport-sample/README.md

```diff
@@ -0,0 +1,20 @@
+# Multiple Viewport Sample
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This sample demonstrates displaying two viewports displaying the same 3d models.
+
+## Purpose
+
+The purpose of this sample is to demonstrate the following:
+
+* The logic needed to balance multiple viewports.
+* How to use the Two-way viewport sync feature.
+
+## Description
+
+When views are synchronized any changes to the properties [ViewState](https://www.imodeljs.org/reference/imodeljs-frontend/views/viewstate/) of either viewport will be immediately applied to the other.  This is done by calling `connect` on [TwoWayViewportSync](https://www.imodeljs.org/reference/imodeljs-frontend/views/twowayviewportsync). One intended use of this feature is for comparing different changesets of the same IModel. Using it that way has the potential for showing the changes over time.
+Note: [TwoWayViewportSync](https://www.imodeljs.org/reference/imodeljs-frontend/views/twowayviewportsync) is currently in beta.
```

Comment:
Maybe this could be worded "At the time this sample was written, TwoWayViewportSync was in beta". So it will be correct in the future if the class becomes public without us having to remember to change this text. Or just remove the note.